index.js

```diff
@@ -5,14 +5,35 @@ console.log("Here's your header:", header)
 
 /***** Deliverable 2 *****/
 
+header.style.color = "blue"
 
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+console.log('This is what the traveler object looks like: ', traveler)
+
+const profileDiv = document.querySelector("#profile")
+profileDiv.innerHTML = `
+    <img src = ${traveler.photo} alt = ${traveler.name} >
+    <h2> ${traveler.name} </h2>
+    <em> ${traveler.nickname} </em>
+`
+
 
 
 /***** Deliverable 4 *****/
 
+const animalsUl = document.querySelector("#animals")
+animalsUl.innerHTML = 
+    traveler.animalSightings.forEach(function(sighting)) {
+        <li data-id = "${sighting.id}">
+        <p>${sighting.description}</p>
+        <img src="${sighting.photo}" alt="${sighting.species}"/>
+        <a href="${sighting.link}" target="_blank">Here's a video about the ${sighting} species!</a>
+        </li>
+    }
 
 
 /***** Deliverable 5 *****/
```

Comment:
**Here's a hint for Deliverable 5**

Check out [this reference](https://www.w3schools.com/cssref/css_selectors.asp) for info on CSS selectors and look into the `remove` method. How can we find the element we need to remove?